esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3748,8 +3748,7 @@ void CWsWuFileHelper::cleanFolder(IFile* folder, bool removeFolder)
     ForEach(*iter)
     {
         OwnedIFile thisFile = createIFile(iter->query().queryFilename());
-        if (thisFile->isFile() == fileBool::foundYes)
-            thisFile->remove();
+        thisFile->remove();
```

Comment:
future/separate JIRA: cleanFolder/createWorkingFolder should be revisited.

Temp. folder should be unique and therefore clean at start of its use by implication.
The call to recursiveRemoveDirectory after it's finished with should be enough.

Reply:
Will create another JIRA

Reply:
https://track.hpccsystems.com/browse/HPCC-27618